tests/testthat/test-add-tbl.R

```diff
@@ -71,28 +67,16 @@ test_that("dm_add_tbl() works", {
   )
 
   # error in case table srcs don't match
-  if (length(test_srcs) > 1) {
-    d1_db <- d1_src[-1]
-    walk(
-      d1_db,
-      ~ expect_dm_error(
-        dm_add_tbl(dm_for_filter, .),
-        "not_same_src"
-      )
-    )
-  }
+  expect_dm_error(
+    dm_add_tbl(dm_for_filter, d1_sqlite),
+    "not_same_src"
+  )
 
   # adding tables to an empty `dm` works for all sources
-  walk(
-    data_1_src,
-    function(data_1) {
-      expect_equal(
-        dm_add_tbl(dm(), test = data_1)$test %>%
-          collect(),
-        data_1 %>%
-          collect()
-      )
-    }
+  expect_equal(
```

Comment:
Use `expect_equivalent_tbl()` here?

Seems worthwhile to go over all tests once more and use new expectations and idioms we implement along the way.